mso/resource_mso_schema_template_anp_epg.go

```diff
@@ -181,7 +242,68 @@ func resourceMSOSchemaTemplateAnpEpgImport(d *schema.ResourceData, m interface{}
+									map_service["deployment_type"] = "cloud_native"
+								} else if models.StripQuotes(servicesCont.S("deploymentType").String()) == "CloudNativeManaged" {
+									map_service["deployment_type"] = "cloud_native_managed"
+								} else if models.StripQuotes(servicesCont.S("deploymentType").String()) == "Third-party" {
+									map_service["deployment_type"] = "third_party"
+								}
+
+								if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-ApiManagement" {
+									map_service["service_type"] = "azure_api_management_services"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-CosmosDB" {
+									map_service["service_type"] = "azure_cosmos_db"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-Databricks" {
+									map_service["service_type"] = "azure_databricks"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-SqlServer" {
+									map_service["service_type"] = "azure_sql"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-Storage" {
+									map_service["service_type"] = "azure_storage"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-StorageBlob" {
+									map_service["service_type"] = "azure_storage_blob"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-StorageFile" {
+									map_service["service_type"] = "azure_storage_file"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-StorageQueue" {
+									map_service["service_type"] = "azure_storage_queue"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-StorageTable" {
+									map_service["service_type"] = "azure_storage_table"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-AksCluster" {
+									map_service["service_type"] = "azure_kubernetes_services"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-ADDS" {
+									map_service["service_type"] = "azure_ad_domain_services"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-ContainerRegistry" {
+									map_service["service_type"] = "azure_contain_registry"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-KeyVault" {
+									map_service["service_type"] = "azure_key_vault"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Azure-Redis" {
+									map_service["service_type"] = "redis_cache"
+								} else if models.StripQuotes(servicesCont.S("serviceType").String()) == "Custom" {
+									map_service["service_type"] = "custom"
+									map_service["custom_service_type"] = models.StripQuotes(servicesCont.S("customSvcType").String())
```

Comment:
I think we can use a map / dict for doing this and not have to use a ton of if/else.